libs/designsystem/src/lib/components/form-field/input-counter/input-counter.component.ts

```diff
@@ -8,6 +8,7 @@ import { TextareaComponent } from './../textarea/textarea.component';
 @Component({
   selector: 'kirby-input-counter',
   templateUrl: './input-counter.component.html',
+  changeDetection: ChangeDetectionStrategy.Default,
```

Comment:
Why default?

Reply:
I think I'll remove this. It was done because I know for a fact that OnPush breaks this component, but this is a weird way of communicating that, I realise 🤔

Reply:
Ah i see - then i actually think it makes sense to set the changeDetection explicitly if it is backed up by a comment (alternatively - just leave a comment).